webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/SrvGrid.scala

```diff
@@ -177,7 +179,7 @@ class SrvGrid(
         JsRaw(s"""
           nodeIds = ${nodeIds}
           reloadTable("${tableId}");
-      """)
+      """) // JsRaw ok, escaped
```

Comment:
It looks sad because `nodeIds` is a `JsArray` but Lift does not escape it by default (it only does `.mkString("[", ", ", "]")`)... we would need to escape it here